testresults/src/org/labkey/testresults/TestResultsController.java

```diff
@@ -687,22 +688,23 @@ public ModelAndView getView(Object o, BindException errors) throws Exception
                 Table.delete(TestResultsSchema.getTableInfoHangs(), filter);
                 Table.delete(TestResultsSchema.getTableInfoTestRuns(), rowId); // delete run last because of foreign key
                 transaction.commit();
+            } catch (Exception x) {
+                response.put("success", false);
```

Comment:
Logging this on the server side can sometimes be helpful to capture the full error info, including its stack trace. But this really depends on what types of exceptions you might expect